pkg/action/install.go

```diff
@@ -766,5 +766,5 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 		atVersion = fmt.Sprintf(" at version %q", version)
 	}
 
-	return filename, errors.Errorf("failed to download %q%s", name, atVersion)
+	return filename, errors.Errorf("failed to download %q%s. Use --devel for a development version", name, atVersion)
```

Comment:
With a few exceptions, we try to avoid avoid referring to Helm CLI commands in `pkg/action`. This is because `pkg/action` can be used by other programs as a library, so asking the user to run certain commands or feature flags may be completely irrelevant to the program.

Reply:
Hi, Okay .. I will do that and update the PR